spec/factories/chargeback_rate.rb

```diff
@@ -15,6 +16,20 @@
       end
     end
 
+    trait :with_custom_compute_details do
+      after(:create) do |chargeback_rate, evaluator|
+        evaluator.detail_params.each do |factory_name, chargeback_rate_params|
```

Comment:
actually yes, It is true, trait `:with_custom_compute_details` is not needed here.
=> removed 

thanks